README.rst

```diff
@@ -1,22 +1,25 @@
-``zope.component``
-==================
+.. image:: https://github.com/zopefoundation/zope.component/actions/workflows/tests.yml/badge.svg
```

Comment:
Why are all these moving above the title? Won't that look very weird rendered on PyPI?

Reply:
Honestly, I'm just doing it the way I'm used to from all other packages I deal with. Like an unwritten convention. It doesn't really matter. If you don't like it I'll move it under the title.